src/pages/discover.js

```diff
@@ -102,44 +151,67 @@ const DiscoverPage = ({ router }) => {
-                  {tagOptions.map(tag => (
-                    <option key={tag} value={tag}>
-                      {tag}
-                    </option>
-                  ))}
-                </select>
-              </Flex>
+            <Flex justifyContent="space-evenly" flexWrap="wrap" mb={4} width="100%">
+              <NavList as="ul" p={0} m={0} justifyContent="space-around" css="margin-right: 256px;">
+                <Box as="li" px={3}>
+                  <Link href="/discover" passHref scroll={false}>
+                    <Nava>
+                      <FormattedMessage id="menu.allCollectives" defaultMessage="All collectives" />
+                    </Nava>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=open%20source" passHref scroll={false}>
+                    <Nava>
+                      <FormattedMessage id="menu.openSourceCollectives" defaultMessage="Open source collectives" />
+                    </Nava>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=open%20source" passHref scroll={false}>
+                    <Nava>
+                      <FormattedMessage id="menu.pledgedCollective" defaultMessage="Pledged collectives" />
+                    </Nava>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=other" passHref scroll={false}>
+                    <Nava>
+                      <FormattedMessage id="menu.others" defaultMessage="Others" />
+                    </Nava>
+                  </Link>
+                </Box>
+              </NavList>
+              <NavList as="ul" p={0} m={0} justifyContent="space-around" css="margin: 0;">
+                <Box as="li" px={3}>
+                  <SelectWrapper name="show" id="show" value={show} onChange={onChange}>
+                    {tagOptions.map(tag => (
+                      <option key={tag} value={tag}>
+                        {tag}
+                      </option>
+                    ))}
+                  </SelectWrapper>
+                </Box>
+              </NavList>
             </Flex>
 
             {collectives && (
               <Fragment>
                 <Flex flexWrap="wrap" width={1} justifyContent="center">
-                  {collectives.map(c => (
-                    <Flex key={c.id} width={[1, 1 / 2, 1 / 4]} mb={3} justifyContent="center">
-                      <CollectiveCard collective={c} LoggedInUser={LoggedInUser} />
-                    </Flex>
-                  ))}
+                  {collectives.map(c => {
+                    return (
+                      <Flex key={c.id} width={[1, 1 / 2, 1 / 4]} mb={3} justifyContent="center">
+                        <CollectiveCard collective={c} LoggedInUser={LoggedInUser} />
```

Comment:
I think this misses a condition to pick between `CollectiveCard` and `PledgedCollectiveCard`. As discussed previously you can assume that a pledged collective will have `isActive` === false